tutorials/_toc.yml

```diff
@@ -5,7 +5,9 @@ format: jb-book
 root: intro
 parts:
   - chapters:
-      - file: introduction/getting-started.md
+      - file: marble/jupyterlab-howto.md
+      - file: marble/version-control.md
```

Comment:
This file isn't in this PR

Reply:
jupyterlab-howto.md is the renamed version of jupyterlab.md. It should be there but I'll take a look.

Reply:
This comment is about line 9.... the version control file is not in this PR

Reply:
Oh, I see.  I've removed that.  It shouldn't be in this PR.